homeassistant/components/enphase_envoy/manifest.json

```diff
@@ -6,7 +6,7 @@
   "documentation": "https://www.home-assistant.io/integrations/enphase_envoy",
   "iot_class": "local_polling",
   "loggers": ["pyenphase"],
-  "requirements": ["pyenphase==1.15.2"],
+  "requirements": ["pyenphase==1.16.0"],
```

Comment:
Can we bump in a separate PR?

Reply:
I check manifest history and there's multiple PR on functions that include the bump. Hence I choose the same.